com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1324,7 +1321,6 @@ public void scanJavaClasses() throws IOException, CompileExceptionError {
         createClassLoaderScanner();
         registerPipelinePlugins();
         scan(scanner, "com.dynamo.bob");
-        scan(scanner, "com.dynamo.bob.pipeline");
```

Comment:
this is just a sub-namespace of `com.dynamo.bob` - we don't want to scan it twice